localstack-core/localstack/services/events/models.py

```diff
@@ -65,6 +117,56 @@ def __post_init__(self):
 RuleDict = dict[RuleName, Rule]
 
 
+@dataclass
+class Replay:
+    name: str
+    region: str
+    account_id: str
+    event_source_arn: Arn
+    destination: ReplayDestination  # Event Bus Arn or Rule Arns
+    event_start_time: Timestamp
+    event_end_time: Timestamp
+    description: Optional[ReplayDescription] = None
+    state: Optional[ReplayState] = None
+    state_reason: Optional[ReplayStateReason] = None
+    event_last_replayed_time: Optional[Timestamp] = None
+    replay_start_time: Optional[Timestamp] = None
+    replay_end_time: Optional[Timestamp] = None
+
+    @property
+    def arn(self) -> Arn:
+        return events_replay_arn(self.name, self.account_id, self.region)
+
+
+ReplayDict = dict[ReplayName, Replay]
+
+
+@dataclass
+class Archive:
+    name: ArchiveName
+    region: str
+    account_id: str
+    event_source_arn: Arn
+    description: ArchiveDescription = None
+    event_pattern: EventPattern = None
+    retention_days: RetentionDays = None
+    state: ArchiveState = ArchiveState.DISABLED
+    creation_time: Timestamp = None
+    size_bytes: int = 0  # TODO how to deal with updating this value?
```

Comment:
currently, there is no clear method for estimating the correct size of an archive to reach parity with aws

Reply:
AWS documents it here: https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-putevent-size.html
Can you create a backlog item and link this there?